deployment/publisher_headless/traffic.py

```diff
@@ -6,92 +7,137 @@
 import aiohttp
 import argparse
 import socket
+import logging
 
+logging.basicConfig(level=logging.INFO)
 
-async def check_dns_time(node: str) -> str:
+
+async def check_dns_time(service: str) -> (str, str):
```

Comment:
A useful tool in these kind of scenarios are [namedtuples](https://docs.python.org/3/library/collections.html#collections.namedtuple): Just a tuple but with named attributes. You can use the name as accessor for the attributes, and they also serve as documentation. Just letting you know in case you weren't aware of them and it makes sense to use them here (or in the future).

Reply:
Yes! But again, as this is another script in an isolated image, didn't want to "boilerplate" too much the code.